### PR TITLE
Node module contract imports

### PR DESCRIPTION
**Pull request by mpetrunic**
Should resolve issue #22

### CODE REVIEW

**Comment by mpetrunic** [comment]
@Latrasis Could you please check it out and publish new version of plugin? :)

File: lib/compiler.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 const fs = require('fs')
 const solc = require('solc')
 const path = require('path')
+var findNodeModules = require('find-node-modules');
```
Comment (Latrasis) [inline lib/compiler.js]: I would be reluctant to include utility dependencies, is there any way to replace this with a shim?

File: lib/compiler.js [hunk 2]
```diff
@@ -35,7 +36,15 @@ function parseErrors(dirPath, errors) {
 function findImports(dirPath) {
   return function(file) {
     try {
-      return {contents: fs.readFileSync(path.resolve(dirPath, file), 'utf8')}
+      var filePath = path.resolve(dirPath, file);
+      // If file doesn't exist try in project node modules
+      if(!fs.existsSync(filePath)) {
+        filePath = path.resolve(
+          findNodeModules({cwd: dirPath, relative: false})[0],
```
Comment (Latrasis) [inline lib/compiler.js]: I would offer to use [`fs#readDirSync`](https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options) instead

File: package.json [hunk 3]
```diff
@@ -31,5 +32,8 @@
   },
   "package-deps": [
     "linter"
-  ]
+  ],
+  "devDependencies": {
+    "openzeppelin-solidity": "^1.10.0"
```
Comment (Latrasis) [inline package.json]: Please remove

**Review by Latrasis — CHANGES_REQUESTED** [review]
Sorry for the delay, thank you for your work! Just some minor nitpicks, looks good.

File: lib/compiler.js (hunk 1 above)
Comment (mpetrunic) [reply lib/compiler.js]: Probably with some recursive searching upwards. I will try, I needed a quick fix for my current project so this was fastest.

File: lib/compiler.js (hunk 2 above)
Comment (mpetrunic) [reply lib/compiler.js]: Yeah sure completely forgot about that.

File: package.json (hunk 3 above)
Comment (mpetrunic) [reply package.json]: It's quite useful for testing purposes and I believe atom will not install dev dependencies. I've added open zeppelin contract from node_modules into sample.sol

**Comment by mpetrunic** [comment]
@Latrasis Sorry for the delay, had some work related stuff to do :) Here is everything rewritten without utility libs. I also added support from imports installed using ethPM (installed_contracts directory)

**Pull request merged**